hq/app/schedule/IamFlaggedUsers.scala

```diff
@@ -18,7 +18,7 @@ object IamFlaggedUsers extends Logging {
     allCreds.map { case (awsAccount, maybeReport) => maybeReport match {
       case Left(error) =>
         error.failures.foreach { failure =>
-          val errorMessage = s"failed to collect credentials report for IAM notifier: ${failure.friendlyMessage}"
+          val errorMessage = s"failed to collect credentials report display for ${awsAccount.name}: ${failure.friendlyMessage}"
```

Comment:
Is it worth leaving `IAM` in the message somewhere to explicitly differentiate from the GCP creds report?
```suggestion
          val errorMessage = s"failed to collect IAM credentials report display for ${awsAccount.name}: ${failure.friendlyMessage}"
```

Reply:
Thanks for taking a look at this @jfsoul. My understanding is that there's no need, because a `CredentialReportDisplay` is made up of `IamUser`s, so it's AWS specific.